util/shell_exec.go

```diff
@@ -57,7 +57,7 @@ func PassthruCommand(cmd *exec.Cmd) (exitCode int) {
 	cmd.Stdout = os.Stdout
 	cmd.Stdin = os.Stdin
 
-	bin := Executor{cmd}
+	bin := Convert(cmd)
```

Comment:
Where is Convert defined?

Reply:
It's in util. I don't remember when we added it but I figured if we already have a sort of "factory method" for this, might as well use it in more than one place.